### PR TITLE
CC-30448: Integration CC-24108

### PR DESCRIPTION
**Pull request by michbeck**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/CC-30448

###### Optional

- Core PR: https://github.com/spryker/suite-nonsplit/pull/7996

### CODE REVIEW

File: src/src/Pyz/Zed/Stock/Persistence/Propel/Schema/spy_stock.schema.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\Stock\Persistence" package="src.Orm.Zed.Stock.Persistence">
+
+    <table name="spy_stock">
+        <behavior name="event">
+            <parameter name="spy_stock_is_active" column="is_active"/>
+        </behavior>
+    </table>
+
+    <table name="spy_stock_store">
+        <behavior name="\Spryker\Zed\AclEntity\Persistence\Propel\Behavior\AclEntityBehavior"/>
```
[the remark this answers is not in the document]
Comment (michbeck) [reply src/src/Pyz/Zed/Stock/Persistence/Propel/Schema/spy_stock.schema.xml:11]: Just wondering, while its already part of master. Shouldn't it be integrated in this case already? @artem-png can you clarify please?

[the remark this answers is not in the document]
Comment (artem-png) [reply src/src/Pyz/Zed/Stock/Persistence/Propel/Schema/spy_stock.schema.xml:11]: For current integration ACL behaviour is not needed. I'd recommend not to integrate something which we don't know

[the remark this answers is not in the document]
Comment (romanhavrylko) [reply src/src/Pyz/Zed/Stock/Persistence/Propel/Schema/spy_stock.schema.xml:11]: @michbeck, it seems like it should be, but some parts (e.g. schema updates) are missing, so probably we need to check what else is missing or why it wasn't integrated. Maybe the better option would be to introduce a separate issue to work on it.

[the remark this answers is not in the document]
Comment (michbeck) [reply src/src/Pyz/Zed/Stock/Persistence/Propel/Schema/spy_stock.schema.xml:11]: @romanhavrylko Already on the way to clarify the situation. Imho @sakharova-yuliia should know something about this changes.

**Review by romanhavrylko — APPROVED** [review]

**Review by abitskil — APPROVED** [review]

File: src/src/Pyz/Zed/Stock/Persistence/Propel/Schema/spy_stock.schema.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```
Comment (artem-png) [inline src/src/Pyz/Zed/Stock/Persistence/Propel/Schema/spy_stock.schema.xml:1]: wrong file directory

**Review by abitskil — APPROVED** [review]

**Pull request merged**